Foodhut/src/sections/introduction.html

```diff
@@ -34,9 +36,9 @@ <h1 class="header-introduction">
       </include>
       <div class="watch-video-container">
         <button class="play-button" type="button">
-          <img src="./assets/images/play-button.svg" alt="play button"/>
+          <div class="play-icon"><img src="./assets/images/play-button.svg" alt="play button"/></div>
```

Comment:
Margin should not be in .play-button. Should be in parent

Reply:
I fixed it, thanks for your contribution

Reply:
Got it

Reply:
It's worked. Thanks